File-Formats/XlsIO/Working-with-Cell-or-Range-Formatting.md

```diff
@@ -2009,7 +2009,7 @@ A complete working example to apply cell text alignment in C# is present on [thi
 
 The following screenshot is the output of previous code:
 
-![Excel document with different alignment options](Working-with-Cell-or-Range-Formatting_images/Working-with-Cell-or-Range-Formatting_img3.jpeg)
+![Excel formatting document with different alignment options](Working-with-Cell-or-Range-Formatting_images/Excel-Formatting-different-alignment-option.jpeg)
```

Comment:
Looks like `Excel formatting document` is not the correct word

Reply:
changed as Excel formatting alone  as keyword is Excel formatting